### PR TITLE
Migrate advanced individual search away from gumby, build foundation for advanced sighting search 

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/components/dataDisplays/IndividualsDisplay.jsx [hunk 1]
```diff
@@ -25,51 +26,64 @@ export default function IndividualsDisplay({
...
+          const adoptionName = deriveIndividualName(
+            individual,
+            'AdoptionName',
+            '-',
+          );
+          return <Text variant="body2">{adoptionName}</Text>;
+        },
       },
     },
+    // {
```
Comment (brmscheiner) [inline src/components/dataDisplays/IndividualsDisplay.jsx:60]: commented out a bunch of features in the individuals table that we dont currently have the response for in elasticsearch. these will hopefully reappearing soon

File: src/models/individual/useIndividualSearchSchemas.js [hunk 2]
```diff
@@ -1,118 +1,104 @@
...
-      labelId: 'HAS_MEDIA',
-      descriptionId: 'HAS_MEDIA_INDIVIDUALS_DESCRIPTION',
-      category: categories.attributes.name,
-      fieldType: fieldTypes.boolean,
-      defaultValue: null,
+      id: 'lastSeen',
+      labelId: 'LAST_SIGHTING_DATE_RANGE',
+      FilterComponent: DateRangeFilter,
+      filterComponentProps: {
+        queryTerm: 'last_sighting',
```
Comment (Emily-Ke) [inline src/models/individual/useIndividualSearchSchemas.js:90]: My documents have a `last_seen` property rather than a `last_sighting` property.

**Review by Emily-Ke — COMMENTED** [review]
Looks ok as a first step in the transition.

File: src/models/individual/useIndividualSearchSchemas.js [hunk 3]
```diff
@@ -1,118 +1,104 @@
...
-      name: 'birthDateRange',
-      labelId: 'BIRTH_DATE_RANGE',
-      descriptionId: 'BIRTH_DATE_RANGE_DESCRIPTION',
-      category: categories.time.name,
-      fieldType: fieldTypes.daterange,
-      defaultValue: [null, null],
+      id: 'firstName',
+      labelId: 'INDIVIDUAL_NAME',
+      FilterComponent: SubstringFilter,
+      filterComponentProps: { queryTerms: ['firstName'] },
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/models/individual/useIndividualSearchSchemas.js:35]: yeah, good call. adding these everywhere

File: src/models/individual/useIndividualSearchSchemas.js (hunk 2 above)
Comment (brmscheiner) [reply src/models/individual/useIndividualSearchSchemas.js:90]: oops nice catch!

File: src/models/sighting/useSightingSearchSchemas.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import SubstringFilter from '../../components/filterFields/SubstringFilter';
+import DateRangeFilter from '../../components/filterFields/DateRangeFilter';
+
+export default function useSightingSearchSchemas() {
+  return [
+    {
+      id: 'owner',
+      labelId: 'OWNER',
+      FilterComponent: SubstringFilter,
+      filterComponentProps: { queryTerms: ['owners.full_name'] },
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/models/sighting/useSightingSearchSchemas.js:11]: actually this one is correct! dont know how multiple owners works but it is indeed `owners.full_name`

**Review by Emily-Ke — APPROVED** [review]

**Pull request merged**